packages/editor/src/components/commands/index.js

```diff
@@ -89,8 +89,7 @@ function useEditorCommandLoader() {
 		name: 'core/open-shortcut-help',
 		label: __( 'Keyboard shortcuts' ),
 		icon: keyboard,
-		callback: ( { close } ) => {
-			close();
```

Comment:
Do we want to keep the Command Palette modal open alongside the shortcuts and preferences modal? Most of the commands I checked call the close callback.

Reply:
Well, if this fix is working then they can’t be open alongside each other. It doesn’t hurt to keep calling the `close` callback but I figured why not remove it.